__tests__/exercise-1.test.js

```diff
@@ -58,10 +52,46 @@ describe("tekCamp()", () => {
 });
 
 
+// YOUR CODE HERE.  follow the template below to flush out your tests.
+
+describe("farenheitCelsius()", () => {
+
+    test("farenheitCelsius(212) should return 100 celsius", () => {
```

Comment:
I like how you use test statements here to solve these problems, very unique approach since we haven't covered these concepts yet!